zoo-css-challenge/style.css

```diff
@@ -176,8 +224,9 @@ footer {
 /*Membership*/
 
 .membership {
-  text-align: center;
+  text-align: auto;
   background-color: #fffffc;
+  max-width:700px;
 }
 
```

Comment:
```
.membership {
  text-align: center;
  background-color: #fffffc;
}

.membership p {
  margin: auto;
  max-width:700px;
}
```

Reply:
thank you for the feedback . i will make sure to look at them again